### PR TITLE
Update `CONF_VERSION` to follow honister

### PR DESCRIPTION
**Pull request by Louson**
Since honister, the `local.conf.sample` bumped. This is needed to pass sanity
checks.

### CODE REVIEW

**Comment by Louson** [comment]
But maybe that would require a different branch

**Comment by Louson** [comment]
An other solution would be to copy the `local.conf.sample` and append the modifications

**Comment by cpb-** [comment]
I wonder if setting `CONF_VERSION = "2"` may break something when building older branches of Poky (especially Dunfell)
I'll run a test in a moment.

**Comment by cpb-** [comment]
Trying to build a not-so-old version of Poky (branch `dunfell` tag `yocto-3.1.13`) with `CONF_VERSION = "2"` in `local conf` fails with the following error.

```
Exception: NotImplementedError: Your version of local.conf was generated from an older/newer version of
local.conf.sample and there have been updates made to this file. Please compare the two
files and merge any changes before continuing.
```

I think we need to parse `local.conf.sample` for Poky-based distro (and update the upcoming PR for non-Poky-based ones).

**Comment by Louson** [comment]
I would say it's not so dependent on Poky. The version is a bump in openembedded-core. Then it probably depends more on the yocto project release. Then, it will be the same for any distribution, or not ?

**Comment by cpb-** [comment]
Yes, it's just that the `local.conf.sample` file to be parsed for `CONF_VERSION` line is not at the same place in Poky and in Arago for example. It's not complicated but it must be taken in account for PR #98.

**Comment by Louson** [comment]
Are you saying you would like to parse the `local.conf.sample` before creating your own `local.conf` ?

What if in the future some versions are not compatible ?
Just for the example, if you need to append a variable in local.conf, will you use `_append` or `:append` ?

Maybe an other solution is to leave the menu manage the version

**Comment by cpb-** [comment]
I think that `cooker` has to extract the `CONF_VERSION` line from `local.conf.sample` file as a default value.

And it could be overridden by the user in the `local.conf` section of the menu.

The `:append` or `_append` choice has to be coherent with the Poky branch indicated in `sources` section. It's the user's responsibility.

**Comment by Louson** [comment]
ok

File: cooker/cooker-menu-schema.json [hunk 1]
```diff
@@ -95,9 +103,16 @@
...
+			    "anyOf": [
+				{
+				    "type": "string",
+				    "minLength": 1
+				},
+				{
+				    "$ref": "#/definitions/targets"
+				}
+			    ]
+			},
```
Comment (pboettch) [inline cooker/cooker-menu-schema.json:115]: No tab in JSON, please.

**Pull request closed without merging**